prow/cluster/jobs/IBM/ibm-licensing-operator/IBM.ibm-licensing-operator.release-4.0.yaml

```diff
@@ -0,0 +1,276 @@
+presubmits:
+  IBM/ibm-licensing-operator:
+  - name: check_ibm-licensing-operator
+    cluster: default
+    branches:
+    - ^release-4.0$
```

Comment:
let's use generic one -^release-.*$ so we can use this prow setup for future release branches